intelmq/bin/intelmqctl.py

```diff
@@ -709,6 +747,10 @@ def list(self, kind=None):
             return self.list_queues()
         elif kind == 'bots':
             return self.list_bots()
+        elif kind == 'queues-and-status':
+            q = self.list_queues()
+            b = self.botnet_status()
+            return q[0] + b[0], [q[1], b[1]]
```

Comment:
Summing the return values is not correct, as 1 and 1 should be 1. Using `max()` or the logical or `|` would work better.

Reply:
Thanks for the hint. (I did the same (`|`) in the `bot_restart` method that we merged few days ago.)